assets/scss/components/stencil/cart/_cart.scss

```diff
@@ -541,12 +546,35 @@ $card-preview-zoom-bottom-offset:       6rem;
 
 .cart-additionalCheckoutButtons {
     @extend %additionalCheckoutButtons;
+    
+    @media (max-width: $screen-small) {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        > div {
+            width: 100% !important;
+
+            p {
+                text-align: center !important;
+            }
+        }
+    }
 }
 
 .previewCart-additionalCheckoutButtons {
-    @extend %additionalCheckoutButtons;
-    padding-right: spacing('single');
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0 (spacing('half') + spacing("quarter"));
     padding-bottom: spacing('single');
+    text-align: center;
+
+    @media (max-width: $screen-small) {
+        > div {
+            width: 100% !important;
```

Comment:
Could we move from important usage?

Reply:
@BC-tymurbiedukhin "!important" directive will set the closest child div to 100% width and overlap the possible container width styles that we get for each of the buttons.
This way we can be sure that at the first level of nesting the buttons are guaranteed to be 100% width.